editor/editor.css

```diff
@@ -149,11 +157,26 @@
     /* justify-content: center; */
 }
 
-.side-panel .side-panel-button-not-active {
+.side-subpanel .side-subpanel-button {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-size: 6px;
```

Comment:
For the readability, it might be better to use bigger size icons and labels for subpanel buttons. We can even use the same size as the upper-level icons/labels. I think `9px` for regular buttons is already small enough, so we may not want to use smaller fonts.